tendrl/commons/objects/__init__.py

```diff
@@ -28,6 +28,7 @@ def thread_safe_method(self, *args, **kws):
 @six.add_metaclass(abc.ABCMeta)
 class BaseObject(object):
     def __init__(self, *args, **kwargs):
+        self._ttl = None
```

Comment:
Wouldn't it affect all other objects?

Reply:
The variable names are different, all objects use "ttl", its only now we are storing the "ttl" passed to save as self._ttl  (there's an underscore).

So this wont affect any object except the NodeContext for now

Reply:
ok. otherwise looks fine to me